frappe/public/js/frappe/list/list_view.js

```diff
@@ -149,6 +149,7 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 			this.get_fields_in_list_view(),
 			[this.meta.title_field, this.meta.image_field],
 			(this.settings.add_fields || []),
+			(this.settings.imageview_fields || []),
```

Comment:
Use `settings.add_fields`, no need to maintain it in another property.

Reply:
okay!

Reply:
People will want to customize this... just show "in_list_view" fields

Reply:
@rmehta Yeah, will make the appropriate changes